src/plugin.rs

```diff
@@ -84,7 +84,7 @@ impl Plugin for AtmospherePlugin {
 /// When added, a skybox will be created as a child.
 /// When removed, that skybox will also be removed.
 /// This behaviour can be disabled by turning off the "detection" feature.
-#[derive(Component, Default, Debug, Clone, Copy)]
+#[derive(Component, Default, Debug, Clone)]
```

Comment:
Why remove `Copy` here?

Reply:
Because `RenderLayers` is no longer `Copy`. It uses a smallvec now.